docs/user/auth-resources.md

```diff
@@ -0,0 +1,56 @@
+# Authentication resources
+
+The table below contains the routes, names, resource names, and verbs for flightctl API endpoints:
+
```

Comment:
Should align with K8s's pattern:

- POST, GET, DELETE on `.../$resource` --> verbs create, list, deletecollection
- GET, PUT, PATCH, DELTE on `.../$resource/{name}` --> verbs get, **update**, patch, delete
- PUT, PATCH on `.../$resource/{name}/status` --> verbs **update, patch** on resource name `$resource**/status**` <-- need to have separate authorization on this endpoint